cmd/wire/main.go

```diff
@@ -83,7 +83,26 @@ func packages(f *flag.FlagSet) []string {
 	return pkgs
 }
 
-type genCmd struct{}
+// newGenerateOptions returns an initialized wire.GenerateOptions, possibly
+// with the Header option set.
+func newGenerateOptions(headerFile string) (*wire.GenerateOptions, error) {
+	opts := new(wire.GenerateOptions)
+	if headerFile != "" {
+		f, err := os.Open(headerFile)
```

Comment:
This leaks `f`. Use `ioutil.ReadFile` to get the byte slice.

Reply:
Done.